src/message/message.service.ts

```diff
@@ -27,11 +27,20 @@ export class MessageService {
     });
   }
 
-  update(id: number, updateMessageDto: UpdateMessageDto) {
-    return `This action updates a #${id} message`;
-  }
+  async updateById(id: number, updateMessageDto: UpdateMessageDto) {
+    const message = await this.messageRepository.findOneBy({ id });
+
+    if (!message)
+      throw new HttpException(
+        'Message not found, cannot create note.',
```

Comment:
Will this make the backend crash? I think we maybe should return a 404 : Message not found

Reply:
This gets caught and sent as the response. 400 might not be the right code though, either 400 or 404....

I suppose the request was valid but a message can't be found with the id given. Probably 404?

Reply:
I've added a test for this now

Reply:
Actually I think it should be 400.

If the request is to create a note and the message id provided doesn't exist in the db then it's a bad request surely?

Reply:
Yeah I think both are valid